index.js

```diff
@@ -0,0 +1,34 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let button = document.createElement("button");
+    let ul = document.createElement("ul");
+    button.id = "jokes";
+    button.innerText = "Click for punchline";
+    document.body.appendChild(button);
+    console.log(ul)
```

Comment:
Append the ul to the body as well since you're just creating in on the script file.